Python/Product/Analysis/Parsing/Ast/MemberExpression.cs

```diff
@@ -76,7 +76,11 @@ public sealed class MemberExpression : Expression {
             res.Append('.');
             if (!this.IsIncompleteNode(ast)) {
                 res.Append(this.GetSecondWhiteSpaceDefaultNull(ast));
-                res.Append(this.GetVerbatimImage(ast) ?? _name);
+                if (format.UseVerbatimImage) {
+                    res.Append(this.GetVerbatimImage(ast) ?? _name);
```

Comment:
@DinoV, can you take a look? I'm not sure what the right thing to do here, actually... it seems like we're being somewhat inconsistent already, in that, if verbatim data is not available (regardless of `format`), we will use the mangled name. So if you parse `class Foo: __bar` and then do `ToCodeString` on it, you'll get `class Foo: __Foo_bar`. This fix essentially relies on that behavior, except it changes it to be consistent with `format`. Is that the best approach?